package.json

```diff
@@ -14,7 +14,7 @@
 	"dependencies": {
 		"@financial-times/sass": "^1.31.0",
 		"@swc/cli": "^0.1.24",
-		"@swc/core": "^1.1.39",
+		"@swc/core": "^1.2.22",
```

Comment:
Let's only change stylelint in this pull-request and keep the other dependencies at the same version.

Reply:
ugh, choore